targets/XPlatCppSdk/source/cppsdk/include/playfab/QoS/QoS.h

```diff
@@ -0,0 +1,27 @@
+/// <summary>
+/// Global definitions for PlayFab QoS.
+/// </summary>
+
+#pragma once
+#include <iostream>
+
+// define body for logging or debug output
+#define LOG_QOS(S) /*std::cout << S;*/
+
+namespace PlayFab
+{
+    namespace QoS
+    {
+        // the error code that is returned if a QoS exception was thrown
+        const int QOS_EXCEPTION_ERROR_CODE = -777;
```

Comment:
nit: Should these be static? Or classed? Right now they aren't marked public, but since they are not within a class I think they count as public to the QoS namespace?

With all caps for these names, makes me think they could be #DEFINE's. But that makes the code slightly uglier. They both seem to be very C like in nature (without the need for a class). 

Right now the call syntax would be QoS.QOS_*. My opinion would be to drop the initial QOS_ for each of these consts so we wouldn't need to read it twice (so just QoS.*). 

But the above is only true if you don't have using namespace QoS; (which is nested in PlayFab, would you want to be using namespace PlayFab.QoS? Or separate calls?). Are you expecting this?

Reply:
Could constexpr work here? similar to what you have in QoSResult?